aiida_vasp/parsers/base.py

```diff
@@ -89,6 +89,10 @@ def get_file(self, fname):
         :param fname: name of the file
         :return: absolute path to the retrieved file
         """
+        # Here I treat STDOUT as a special case, since the exact name of the
+        # diverted stdout can be customised in the options of the CalcJobNode
+        if fname == 'STDOUT':
+            fname = self.node.get_option('scheduler_stdout')
```

Comment:
I think this needs to be changed - previous I am parsing the STDOUT from the scheduler which includes that from VASP as well as any output from the scheduler. Now I can see it is done properly by redirecting the STDOUT from the code using `calcinfo.stdout_name`. This means the STDOUT from the scheduler no longer contains those from VASP.

probably `fname = VaspCalculation._VASP_OUTPUT` is the right way?

Reply:
Indeed, this slipped my mind and I discovered it when running live tests as well.